source/p004-style-guide.pug

```diff
@@ -0,0 +1,108 @@
+include modules/head/head
+include modules/heading/heading
+include modules/elements/elements
+include modules/sidebar/sidebar
+include modules/style-guide/style-guide
+
+doctype html
+html
+  head
+    +head(head.p004-style-guide)
+
+  body
+  +style-guide('Fram.io - Demo project', '1.1', '05/10/2017')
+
+    +set-block('changelog')
+      ul.guide-content__changelog
+        li.font-weight-normal 05/10/2017 - V1.1
+          ul.guide-content__changelog__detail
+           li Change font to Muli (Google font support)
```

Comment:
please using data json. after that using loop

Reply:
Updated with data.json and loop